projects/opendr_ws/src/opendr_perception/README.md

```diff
@@ -354,35 +354,39 @@ whose documentation can be found here: [Deep Sort docs](../../../../docs/referen
+```shell
+rosrun opendr_perception panoptic_segmentation_efficient_ps.py
+```
 
-3. Default output topics:
-   - Output images: `/opendr/panoptic/semantic`, `/opendr/panoptic/instance`, `/opendr/panoptic/rgb_visualization`
-   - Detection messages: `/opendr/panoptic/semantic`, `/opendr/panoptic/instance`
+The following optional arguments are available:
+- `-h, --help`: show a help message and exit
+- `--input_rgb_image_topic INPUT_RGB_IMAGE_TOPIC` : listen to RGB images on this topic (default=`/usb_cam/image_raw`)
+- `--checkpoint CHECKPOINT` : download pretrained models [cityscapes, kitti] or load from the provided path (default=`cityscapes`)
+- `--output_rgb_image_topic OUTPUT_RGB_IMAGE_TOPIC`: publish the semantic and instance maps on this topic as `OUTPUT_HEATMAP_TOPIC/semantic` and `OUTPUT_HEATMAP_TOPIC/instance` (default=`/opendr/panoptic`)
+- `--visualization_topic VISUALIZATION_TOPIC`: publish the panoptic segmentation map as an RGB image on `VISUALIZATION_TOPIC` or a more detailed overview if using the `--detailed_visualization` flag (default=`/opendr/panoptic/rgb_visualization`)
+- `--detailed_visualization`: generate a combined overview of the input RGB image and the semantic, instance, and panoptic segmentation maps and publish it on `OUTPUT_RGB_IMAGE_TOPIC` (default=deactivated)
+
+### LiDAR Based Panoptic Segmentation ROS Node
+A ROS node for performing panoptic segmentation on a specified pointcloud stream using the [EfficientLPS](../../../../src/opendr/perception/panoptic_segmentation/README.md) network.
+Assuming that the OpenDR catkin workspace has been sourced, the node can be started with the following sequence.
+
+First one needs to download SemanticKITTI dataset into POINTCLOUD_LOCATION as it is described in the [Panoptic Segmentation Datasets](../../../../src/opendr/perception/panoptic_segmentation/datasets/README.md). Then, once the SPLIT type is specified (train, test or "valid", default "valid"), the point **PointCloud2 Publisher** can be started using the following line:
+```shell
+rosrun opendr_perception point_cloud_2_publisher.py -d POINTCLOUD_LOCATION -s SPLIT
+```
+After starting the **PointCloud2 Publisher**, one can start **EfficientLPS Node** using the following line:
+```shell
+rosrun opendr_perception panoptic_segmentation_efficient_lps.py /opendr/dataset_point_cloud2
+```
 
-   For viewing the output, refer to the [notes above.](#notes)
+The following optional arguments are available:
+- `-h, --help`: show a help message and exit
+- `--input_pcl_topic INPUT_PCL_TOPIC` : listen to pointclouds on this topic (default=`/usb_cam/pcl_raw`)
+- `--checkpoint CHECKPOINT` : download pretrained models [semantickitti, nuscenes (future)] or load from the provided path (default=`semantickitti`)
+- `--output_rgb_visualization_topic OUTPUT_RGB_VISUALIZATION_TOPIC`: publish the rgb visualization on this topic on `OUTPUT_RGB_VISUALIZATION_TOPIC` (default=`/opendr/panoptic`) 
```

Comment:
Since this node has point cloud input it should be moved to the relevant point cloud input section. Nodes are separated by input type.

Moreover, documentation for the new node should follow the format in other nodes as closely as possible. Need to add a description at the start with all relevant links as in other nodes, and format the rest as instructions for basic usage steps, please refer  to other nodes' sections.

Reply:
I have moved this at the bottom of point cloud inputs. I think this should be sufficient enough at the moment. Sorry for the earlier mess, I didn't realize they are categorized according to their input types.

Reply:
Thanks, no problem!